opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/metric_reader.py

```diff
@@ -31,14 +32,26 @@
     ObservableUpDownCounter,
     UpDownCounter,
 )
-from opentelemetry.sdk._metrics.point import AggregationTemporality, Metric
+from opentelemetry.sdk._metrics.point import AggregationTemporality
+from opentelemetry.sdk._metrics.point import Metric as PointMetric
+from opentelemetry.sdk._metrics.point import PointT
 from opentelemetry.sdk.environment_variables import (
     _OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE,
 )
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.util.instrumentation import InstrumentationScope
 
 _logger = getLogger(__name__)
 
 
+@dataclass(frozen=True)
+class Metric:
+    description: str
+    name: str
+    unit: str
+    point: PointT
```

Comment:
This point is single `Sum`, `Gauge`, or `Histogram` record. I was expecting the refactor will match the proto model where the point contains the repeated data points with nested structure. This is still the flat structured and some grouping by point type is done at the exporter.

Reply:
Right, fixing...

Reply:
Fixed